test/parallel/test-http-blank-header.js

```diff
@@ -47,7 +47,9 @@ server.listen(0, common.mustCall(() => {
             '\r\n\r\nhello world'
     );
   }));
-
+  c.on('data', common.mustCall((data) => {
+    assert.strictEqual('HTTP/1.1 400 Bad Request\r\n\r\n', data.toString());
```

Comment:
You should probably accumulate the data in a string here and then perform the assertion in the `'end'` handler.